bazel/repository_locations.bzl

```diff
@@ -1,7 +1,8 @@
 REPOSITORY_LOCATIONS = dict(
+    # can't have more than one comment between envoy line and commit line in 
```

Comment:
The `-A 2` on this line
https://github.com/solo-io/envoy-gloo/blob/46ca9b475584b21655fb40c617769ea870c35838/ci/check_extensions_build_config.sh#L27

Selects how many lines to search for to find the envoy commit

Reply:
Yeah. We can't arbitrarily bump it though because eventually it will be looking at the block for the next dependency right? Unless the envoy dependency is just always listed last but that seems just as arbitrary to me

Reply:
> We can't arbitrarily bump it though because eventually it will be looking at the block for the next dependency right?

yes, that is correct.